### PR TITLE
Minor manual fixes

### PR DESCRIPTION
**Pull request by kroening**
- [ ] Each commit message has a non-empty body, explaining why the change was made.
- n/a Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- n/a The feature or user visible behaviour I have added or modified has been documented in the User Guide in doc/cprover-manual/
- n/a Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- n/a My commit message includes data points confirming performance improvements (if claimed).
- [ ] My PR is restricted to a single feature or bugfix.
- [X] White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

File: doc/cprover-manual/cbmc-tutorial.md [hunk 1]
```diff
@@ -135,18 +135,18 @@ In the example above, we used a program that starts with a `main`
...
-  for(i=0; i<10; i++)
-    sum+=array[i];
+int sum()
+{
+  unsigned i, sum;
 
-  return sum;
+  sum = 0;
+  for(i = 0; i < 10; i++)
+    sum += array[i];
```
Comment (peterschrammel) [inline doc/cprover-manual/cbmc-tutorial.md:150]: Shouldn't this return something?

File: doc/cprover-manual/file2.c [hunk 2]
```diff
@@ -0,0 +1,10 @@
+int array[10];
+
+int sum()
+{
+  unsigned i, sum;
+
+  sum = 0;
+  for(i = 0; i < 10; i++)
+    sum += array[i];
+}
```
Comment (peterschrammel) [inline doc/cprover-manual/file2.c:12]: return ?

**Review by peterschrammel — APPROVED** [review]

**Review by allredj — COMMENTED** [review]
:warning:
This PR failed Diffblue compatibility checks (cbmc commit: 491257d2ed4500117407987da9528d79cfd49cde).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/105517384
Status will be re-evaluated on next push.
Common spurious failures:
  - the cbmc commit has disappeared in the mean time (e.g. in a force-push)
  - the author is not in the list of contributors (e.g. first-time contributors).

  - the compatibility was already broken by an earlier merge.

**Review by allredj — COMMENTED** [review]
:warning:
This PR failed Diffblue compatibility checks (cbmc commit: 4cc9b5d61149c3265a3476c18ca86aeb6d3020f8).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/105517622
Status will be re-evaluated on next push.
Common spurious failures:
  - the cbmc commit has disappeared in the mean time (e.g. in a force-push)
  - the author is not in the list of contributors (e.g. first-time contributors).

  - the compatibility was already broken by an earlier merge.

**Review by allredj — COMMENTED** [review]
:heavy_check_mark:
Passed Diffblue compatibility checks (cbmc commit: de61ff0e5712dd04a19a867283b9e042faf4ecf7).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/105518441

File: doc/cprover-manual/file2.c [hunk 3]
```diff
@@ -7,4 +7,6 @@ int sum()
   sum = 0;
   for(i = 0; i < 10; i++)
     sum += array[i];
+
+  return 0;
```
Comment (tautschnig) [inline doc/cprover-manual/file2.c]: Wouldn't it be more meaningful to return `sum`? (And, as a nit-pick, I don't think this particular change should be in a separate commit, because an earlier commit (wrongly) removed the `return` statement here.)

**Review by allredj — COMMENTED** [review]
:warning:
This PR failed Diffblue compatibility checks (cbmc commit: b44a88034f937e2de2203660bba689039c991bdf).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/105539551
Status will be re-evaluated on next push.
Common spurious failures:
  - the cbmc commit has disappeared in the mean time (e.g. in a force-push)
  - the author is not in the list of contributors (e.g. first-time contributors).

  - the compatibility was already broken by an earlier merge.

**Review by allredj — COMMENTED** [review]
:heavy_check_mark:
Passed Diffblue compatibility checks (cbmc commit: fb3a7f50d0a7b8c893dfcb7683360da76aad1dd9).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/105540489
